Frameworks/UIKit/UIViewController.mm

```diff
@@ -342,14 +342,7 @@ - (CGRect)_orientationRect:(UIInterfaceOrientation)orientation {
     appFrame = [[UIScreen mainScreen] applicationFrame];
 
     if (priv->_presentationStyle == UIModalPresentationFormSheet) {
```

Comment:
Not part of the change, but again should be using accessors for accessing properties.